source/MCR20Drv.c

```diff
@@ -111,7 +111,9 @@ void
     xcvr_spi_configure_speed(gXcvrSpiInstance_c, 8000000);
 
     gXcvrDeassertCS_d();
-    MCR20Drv_RST_B_Deassert();
+    #if !defined(TARGET_KW24D)
```

Comment:
Hi Mac, Can we try running the test with out this commented out for KW24D. I am thinking the issue seen was related to the low-power code and not resetting the modem.